lib/simp/media/type/control-repo.rb

```diff
@@ -151,7 +151,7 @@ def munge_hierayaml(hierayaml)
                 end
               end
               unless found
-                hash = { 'name' => 'SIMP Compliance Engine', 'lookup_key' => 'compliance_markup::enforcement' }
+                hash = {:name => 'SIMP Compliance Engine', :lookup_key => 'compliance_markup::enforcement' }
```

Comment:
Since this is a best-practices or general refactor update, I'll nitpick. You shouldn't have 0 spaces at the beginning after the opening { if you're going to have a single space before the closing }.

Personally, I'd put a space after the opening curly brace. My best guess is that rubocop thinks you need them on new-lines,. In any case, just be consistent.

Reply:
this looks to be an accident, hiera yaml uses string keys not symbols